ts-rs/src/export.rs

```diff
@@ -126,15 +146,22 @@ pub(crate) fn export_type_to<T: TS + ?Sized + 'static, P: AsRef<Path>>(
     Ok(())
 }
 
-/// Returns the generated defintion for `T`.
-pub(crate) fn export_type_to_string<T: TS + ?Sized + 'static>() -> Result<String, ExportError> {
+/// Returns the generated definition for `T`.
+pub(crate) fn export_to_string<T: TS + ?Sized + 'static>() -> Result<String, ExportError> {
     let mut buffer = String::with_capacity(1024);
     buffer.push_str(NOTE);
-    generate_imports::<T::WithoutGenerics>(&mut buffer)?;
+    generate_imports::<T::WithoutGenerics>(&mut buffer, &*default_out_dir())?;
     generate_decl::<T>(&mut buffer);
     Ok(buffer)
 }
 
+pub(crate) fn default_out_dir() -> Cow<'static, Path> {
+    match std::env::var("TS_RS_EXPORT_DIR") {
+        Err(..) => Cow::Borrowed(Path::new("./bindings")),
```

Comment:
Cool, I didn't know `Err(..)` was valid syntax to ignore enum data, I though it had to be `Err(_)`

Reply:
It's a character more, but I kinda like how it looks 😆